graph2tac/tfgnn/dataset.py

```diff
@@ -493,16 +477,16 @@ class DataServerDataset(Dataset):
     proofstate_data_spec = (state_spec, action_spec, graph_id_spec)
     definition_data_spec = (loader_graph_spec, num_definitions_spec, definition_names_spec)
 
-    def __init__(self, data_dir: Path, max_subgraph_size: int = 1024, **kwargs):
+    def __init__(self, data_dir: Path, split = None, split_random_seed = 0, max_subgraph_size: int = 1024, **kwargs):
```

Comment:
Again, I would removed the defaults.  In particular, `split = None` would likely crash the code, right???